phpBB/language/en/email/acp_login.txt

```diff
@@ -0,0 +1,5 @@
+Subject: phpBB.com Admin Control Panel Login
```

Comment:
it would but extensions can only find email templates inside phpBB core right now